src/main/java/org/ihtsdo/buildcloud/core/service/ProductServiceImpl.java

```diff
@@ -357,6 +357,22 @@ private void updateProductBuildConfiguration(final Map<String, String> newProper
 			}
 		}
 
+		if (newPropertyValues.containsKey(ADDITIONAL_PREVIOUS_PUBLISHED_PACKAGES)) {
+			final ReleaseCenter releaseCenter = product.getReleaseCenter();
+			final String additionalPackages = newPropertyValues.get(ADDITIONAL_PREVIOUS_PUBLISHED_PACKAGES);
+			if (StringUtils.isEmpty(additionalPackages)) {
+				configuration.setAdditionalPreviousPublishedPackages(null);
+			} else {
+				String[] previousPublishedPackages = Arrays.stream(additionalPackages.split(",")).map(String::trim).toArray(String[]::new);
+				for (String previousPublishedPackage : previousPublishedPackages) {
+					if (!publishService.exists(product.getReleaseCenter(), previousPublishedPackage)) {
```

Comment:
@QuyenLy87  These additional packages might have not been published by the release center. (i.e Swiss release center doesn't actually publish the common french package)